distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -544,16 +546,43 @@ private static Integer findPortNumber(Integer portToTry) {
     }
 
     protected Option[] configureLogLevel() {
-        return options(when(
-                System.getProperty(AdminConfig.TEST_LOGLEVEL_PROPERTY) != null).useOptions(
-                systemProperty(AdminConfig.TEST_LOGLEVEL_PROPERTY).value(System.getProperty(
-                        AdminConfig.TEST_LOGLEVEL_PROPERTY,
-                        ""))),
-                editConfigurationFilePut("etc/" + LOG_CONFIG_PID + ".cfg",
-                        "log4j.additivity.org.apache.activemq.artemis",
+        final String logLevel = System.getProperty(TEST_LOG_LEVEL_PROPERTY);
+        final String securityLogLevel = System.getProperty(TEST_SECURITY_LOG_LEVEL_PROPERTY);
+        return options(
+                editConfigurationFilePut(LOGGER_CONFIGURATION_FILE_PATH,
+                    "log4j2.rootLogger.level",
+                        DEFAULT_LOG_LEVEL),
+                when(StringUtils.isNotEmpty(logLevel)).useOptions(combineOptions(
+                        createSetLogLevelOption("ddf", logLevel),
+                        createSetLogLevelOption("org.codice", logLevel))),
+                when(StringUtils.isNotEmpty(securityLogLevel)).useOptions(combineOptions(
+                        createSetLogLevelOption("ddf.security.expansion.impl.RegexExpansion", securityLogLevel),
+                        createSetLogLevelOption("df.security.service.impl.AbstractAuthorizingRealm", securityLogLevel))),
+                editConfigurationFilePut("etc/org.ops4j.pax.logging.cfg",
+                        "log4j2.logger.org_apache_activemq_artemis.additivity",
                         "true"));
     }
 
+    /**
+     * Creates options to add log configuration lines to the etc/org.ops4j.pax.logging.cfg file.
+     * See {@see org.apache.karaf.log.core.internal.LogServiceLog4j2Impl}.
+     *
+     * @param name  name of the logger to set
+     * @param level String value to set the logger level
+     * @return options to set the log level
+     */
+    private Option[] createSetLogLevelOption(String name, String level) {
+        final String loggerPrefix = "log4j2.logger.";
+        final String loggerKey = name.replace('.', '_').toLowerCase();
+        return options(
+                editConfigurationFilePut(LOGGER_CONFIGURATION_FILE_PATH,
+                        loggerPrefix + loggerKey + ".name",
```

Comment:
This is pretty clean but I think we usually go with String.format for creating strings like this.